modules/components/pages/inputs/redpanda_common.adoc

```diff
@@ -0,0 +1,284 @@
+
+* `fetch_max_partition_bytes`
+* `fetch_max_bytes`
+
+You can break batches down further using the xref:components:processors/split.adoc[`split`] processor.
+
+== Metadata
+
+This input adds the following metadata fields to each message:
+
+- `kafka_key`
+- `kafka_topic`
+- `kafka_partition`
+- `kafka_offset`
+- `kafka_timestamp`
+- `kafka_timestamp_unix`
+- `kafka_tombstone_message`
+- All record headers
+
+== Fields
+
+=== `topics`
+
+A list of topics to consume from. Use commas to separate multiple topics in a single element.
+
+When a `consumer_group` is specified, partitions are automatically distributed across consumers of a topic. Otherwise, all partitions are consumed.
+
+Alternatively, you can specify explicit partitions to consume by using a colon after the topic name. For example, `foo:0` would consume the partition `0` of the topic foo. This syntax supports ranges. For example, `foo:0-10` would consume partitions `0` through to `10` inclusive.
+
+It is also possible to specify an explicit offset to consume from by adding another colon after the partition. For example, `foo:0:10` would consume the partition `0` of the topic `foo` starting from the offset `10`. If the offset is not present (or remains unspecified) then the field `start_from_oldest` determines which offset to start from.
+
+*Type*: `array`
+
+```yml
+# Examples
+
+topics:
+  - foo
+  - bar
+
+topics:
+  - things.*
+
+topics:
+  - foo,bar
+
+topics:
+  - foo:0
+  - bar:1
+  - bar:3
+
+topics:
+  - foo:0,bar:1,bar:3
+
+topics:
+  - foo:0-5
+```
+
+=== `regexp_topics`
+
+Whether listed topics are interpreted as regular expression patterns for matching multiple topics. When topics are specified with explicit partitions, this field must remain set to `false`.
+
+*Type*: `bool`
+
+*Default*: `false`
+
+=== `rack_id`
+
+A rack specifies where the client is physically located, and changes fetch requests to consume from the closest replica as opposed to the leader replica.
+
+
+*Type*: `string`
+
+*Default*: `""`
+
+=== `start_from_oldest`
+
+Whether to consume from the oldest available offset. Otherwise, messages are consumed from the latest offset. This setting is applied when creating a new consumer group or the saved offset no longer exists.
+
+*Type*: `bool`
+
+*Default*: `true`
+
+=== `fetch_max_bytes`
+
+The maximum amount of bytes that a broker tries to send during a fetch. 
+
+If individual records are larger than the `fetch_max_bytes` value, brokers will still send them.
+
+*Type*: `string`
+
+*Default*: `50MiB`
+
+=== `fetch_min_bytes`
+
+The minimum amount of bytes that a broker tries to send during a fetch. This field is equivalent to the Java setting `fetch.min.bytes`.
+
+*Type*: `string`
+
+*Default*: `1B`
+
+=== `fetch_max_partition_bytes`
+
+The maximum amount of bytes that are consumed from a single partition in a fetch request. This field is equivalent to the Java setting `fetch.max.partition.bytes`.
```

Comment:
```suggestion
The maximum number of bytes that are consumed from a single partition in a fetch request. This field is equivalent to the Java setting `fetch.max.partition.bytes`.
```

Reply:
I believe amount is for items that are uncountable, like raindrops in the sky. Number is for something countable...at least that's how I've always used them.